dex/farm/src/lib.rs

```diff
@@ -107,9 +112,12 @@ pub trait Farm:
         require!(token_in == farming_token_id, "Bad input token");
         require!(enter_amount > 0, "Cannot farm with amount of 0");
 
+        self.farming_token_total_liquidity()
```

Comment:
why is this needed ? Where is this inside the farm contracts ?

Reply:
oh, I see. For the bounded APR computation

Reply:
Yep. It's the only reason it's needed. Couldn't use SC balance directly in case of farming token being the same as the reward token.